### PR TITLE
fix(CurriculumCard): Wrap difficulty in span to avoid translation error

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

Wraps the difficulty within the curriculum card in a span to fix the `insertBefore` errors that are happening when you use Google Translate to translate a page and cause the page to crash. More info about the issue: https://github.com/facebook/react/issues/11538#issuecomment-390386520.

See monolith/portal app PRs linked below for testing instructions.

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [WEB-1949]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/30550)  | [Monolith Env](https://pr-30550-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/1902)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1902)   |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.1b9490.0<br/>@codecademy/gamut-labs@31.11.7-alpha.1b9490.0<br/>@codecademy/styleguide@57.0.6-alpha.1b9490.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://626ffd1a276ce260982d4a3a--gamut-preview.netlify.app>https://626ffd1a276ce260982d4a3a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/626ffd1a276ce260982d4a3a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6270003a459f1a640a9935cd--gamut-preview.netlify.app>https://6270003a459f1a640a9935cd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6270003a459f1a640a9935cd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.b61bf6.0<br/>@codecademy/gamut-labs@31.11.7-alpha.b61bf6.0<br/>@codecademy/styleguide@57.0.6-alpha.b61bf6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://627024be3f572f14a3d263f6--gamut-preview.netlify.app>https://627024be3f572f14a3d263f6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/627024be3f572f14a3d263f6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.f9f9c0.0<br/>@codecademy/gamut-labs@31.11.7-alpha.f9f9c0.0<br/>@codecademy/styleguide@57.0.6-alpha.f9f9c0.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.00c91d.0<br/>@codecademy/gamut-labs@31.11.7-alpha.00c91d.0<br/>@codecademy/styleguide@57.0.6-alpha.00c91d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62702b328d1f330075bf268c--gamut-preview.netlify.app>https://62702b328d1f330075bf268c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62702b328d1f330075bf268c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.6a5c0e.0<br/>@codecademy/gamut-labs@31.11.7-alpha.6a5c0e.0<br/>@codecademy/styleguide@57.0.6-alpha.6a5c0e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://627031097f6bca04f6d7c018--gamut-preview.netlify.app>https://627031097f6bca04f6d7c018--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/627031097f6bca04f6d7c018>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.a2ac6d.0<br/>@codecademy/gamut-labs@31.11.7-alpha.a2ac6d.0<br/>@codecademy/styleguide@57.0.6-alpha.a2ac6d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://627034e61f928d08d66b2200--gamut-preview.netlify.app>https://627034e61f928d08d66b2200--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/627034e61f928d08d66b2200>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.e91f08.0<br/>@codecademy/gamut-labs@31.11.7-alpha.e91f08.0<br/>@codecademy/styleguide@57.0.6-alpha.e91f08.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://627037a3df61500b8394c012--gamut-preview.netlify.app>https://627037a3df61500b8394c012--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/627037a3df61500b8394c012>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.13bc85.0<br/>@codecademy/gamut-labs@31.11.7-alpha.13bc85.0<br/>@codecademy/styleguide@57.0.6-alpha.13bc85.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://627039e6df61500e3794c011--gamut-preview.netlify.app>https://627039e6df61500e3794c011--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/627039e6df61500e3794c011>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.32028b.0<br/>@codecademy/gamut-labs@31.11.7-alpha.32028b.0<br/>@codecademy/styleguide@57.0.6-alpha.32028b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62703e6eecbf1d114cd393f1--gamut-preview.netlify.app>https://62703e6eecbf1d114cd393f1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62703e6eecbf1d114cd393f1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.a684ef.0<br/>@codecademy/gamut-labs@31.11.7-alpha.a684ef.0<br/>@codecademy/styleguide@57.0.6-alpha.a684ef.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62704170e65a4a1438cc0dd4--gamut-preview.netlify.app>https://62704170e65a4a1438cc0dd4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62704170e65a4a1438cc0dd4>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.f4a567.0<br/>@codecademy/gamut-labs@31.11.7-alpha.f4a567.0<br/>@codecademy/styleguide@57.0.6-alpha.f4a567.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://627059b0d0eca42c06e8fd4c--gamut-preview.netlify.app>https://627059b0d0eca42c06e8fd4c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/627059b0d0eca42c06e8fd4c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.118-alpha.59cba4.0<br/>@codecademy/gamut-labs@31.11.7-alpha.59cba4.0<br/>@codecademy/styleguide@57.0.6-alpha.59cba4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62705b1766abd52bf9940daf--gamut-preview.netlify.app>https://62705b1766abd52bf9940daf--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62705b1766abd52bf9940daf>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.119-alpha.3c5043.0<br/>@codecademy/gamut-labs@31.11.8-alpha.3c5043.0<br/>@codecademy/styleguide@57.0.7-alpha.3c5043.0<br/></pre>

File: packages/gamut-labs/src/CurriculumCard/Difficulty/index.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const Difficulty: React.FC<ContentDifficultyProps> = ({
       {times(difficultyNumber + 1, () => (
         <DifficultySpan key={uniqueId()} variant={variant} />
       ))}
-      {DifficultyString[difficultyNumber]}
+      <Box as="span">{DifficultyString[difficultyNumber]}</Box>
```
Comment (MattSchiller) [inline packages/gamut-labs/src/CurriculumCard/Difficulty/index.tsx:47]: Nit: any reason we're not using `<Text>` here instead? Just wondering, semantically

**Review by MattSchiller — APPROVED** [review]
I'm into it. Could you cut a ticket to investigate a linter rule to perhaps find something like this?

Or even better maybe, adding steps to our cypress E2E tests that run pages with a translation extension to get ahead of changes like this before we start introducing non-breaking, edge-case errors like this again?

Thanks for fixing this 👍

File: packages/gamut-labs/src/CurriculumCard/Difficulty/index.tsx (hunk 1 above)
Comment (aresnik11) [reply packages/gamut-labs/src/CurriculumCard/Difficulty/index.tsx:47]: good call, that makes more sense and they're both a `span`

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.119-alpha.257982.0<br/>@codecademy/gamut-labs@31.11.8-alpha.257982.0<br/>@codecademy/styleguide@57.0.7-alpha.257982.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62716f1adbffe3007b11c84d--gamut-preview.netlify.app>https://62716f1adbffe3007b11c84d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62716f1adbffe3007b11c84d>Deploy Logs</a></p>
